test/e2e/framework/volume/fixtures.go

```diff
@@ -620,6 +616,9 @@ func generateWriteFileCmd(content, fullPath string) []string {
 // CheckVolumeModeOfPath check mode of volume
 func CheckVolumeModeOfPath(f *framework.Framework, pod *v1.Pod, volMode v1.PersistentVolumeMode, path string) {
 	if volMode == v1.PersistentVolumeBlock {
+		// NOTE: gcepd-csi doesn't support a block volume in windows, this method won't
```

Comment:
this is a general test library, I'm not sure we need this comment

Reply:
I removed this comment